src/figuresClasses.ts

```diff
@@ -1,11 +1,96 @@
-export interface Figure {}
+export enum Shape {
+  Triangle = 'triangle',
+  Rectangle = 'rectangle',
+  Circle = 'circle',
+}
+
+export type Color = 'red' | 'green' | 'blue';
+
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
+
+const roundeArea = (area: number): number => {
+  return Math.floor(area * 100) / 100;
+};
 
-export class Triangle implements Figure {}
+export class Triangle implements Figure {
+  shape: Shape = Shape.Triangle;
 
-export class Circle implements Figure {}
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (this.a <= 0 || this.b <= 0 || this.c <= 0) {
```

Comment:
```suggestion
    if (a <= 0 || b <= 0 || c <= 0) {
```
You have access to arguments in construcor